src/Functions/changeDate.cpp

```diff
@@ -346,55 +346,97 @@ REGISTER_FUNCTION(ChangeDate)
     {
         FunctionDocumentation::Description description = "Changes the year component of a date or date time.";
         FunctionDocumentation::Syntax syntax = "changeYear(date_or_datetime, value);";
-        FunctionDocumentation::Arguments arguments = {{"date_or_datetime", "The value to change. Type: Date, Date32, DateTime, or DateTime64"}, {"value", "The new value. Type: [U]Int*"}};
-        FunctionDocumentation::ReturnedValue returned_value = "The same type as date_or_datetime.";
+        FunctionDocumentation::Arguments arguments = {
+            {"date_or_datetime", "The value to change.", {"Date", "Date32", "DateTime", "DateTime64"}},
+            {"value", "The new value.", {"(U)Int*"}}
+        };
+        FunctionDocumentation::ReturnedValue returned_value = {"Returns date time with modified year component.", {"Date", "Date32", "DateTime", "DateTime64"}};
```

Comment:
`Returns date time with modified year component.` --> `A value of the same type as date_or_datetime with modified year component`

Please apply the same change also below.